internal/messagebirdtest/test_client.go

```diff
@@ -0,0 +1,51 @@
+package messagebirdtest
+
+import (
+	"crypto/tls"
+	"log"
+	"net"
+	"net/http"
+	"testing"
+
+	messagebird "github.com/messagebird/go-rest-api"
+)
+
+// testWriter can be used to have the client write to the tests's error log.
+type testWriter struct {
+	t *testing.T
+}
+
+// Write logs the provided buffer to the current test's error log.
+func (w testWriter) Write(p []byte) (int, error) {
+	w.t.Logf("%s", p)
+
+	return len(p), nil
+}
+
+// Client initializes a new MessageBird client that uses the
+func Client(t *testing.T) *messagebird.Client {
+	return client(t, "test_gshuPaZoeEG6ovbc8M79w0QyM")
```

Comment:
Is it ok to commit this? Or should we rather set a secret in the CI env?

Reply:
I'll look into how Travis facilitates this, thanks!

Reply:
Hm, it seems it's easily done by encrypting it and setting it as an environment variable. The issue with this is that this won't work for PRs from forks, meaning the build would always fail for (external) contributors: https://docs.travis-ci.com/user/encryption-keys/

Reply:
I have removed the test key from the test client for now, as it is not used at this point anyway.